class.dynamic_cdn.php

```diff
@@ -50,6 +50,7 @@ public function init() {
 				add_filter( 'the_content'/*'dynamic_cdn_content'*/, array( $this, 'filter_uploads_only' ) );
 			} else {
 				add_filter( 'dynamic_cdn_content', array( $this, 'filter' ) );
+				add_filter( 'wp_calculate_image_srcset', array( $this, 'srcsets' ), 10, 5 );
```

Comment:
Would it make sense to add this filter to both parts of the conditional? Or is it intended to be used when _all_ images are filtered and not just uploaded media? If it makes sense for both, maybe bump outside of the conditional entirely so it runs all the time.